src/components/base/JCheckbox/JCheckbox.js

```diff
@@ -1,29 +1,23 @@
-// @flow
+// @flow strict
 
+import classNames from 'classnames'
 import React, { PureComponent } from 'react'
 
-import classNames from 'classnames'
-import {
-  JText, JIcon,
-} from 'components/base'
+import { JIcon } from 'components/base'
+
+import jCheckboxStyles from './jCheckbox.m.scss'
 
-type JCheckboxColor = 'white' | 'gray'
+type JCheckboxHandler = (boolean) => void
 
 type Props = {|
```

Comment:
I guess, checkbox is [Base Component](https://github.com/jibrelnetwork/frontend/blob/master/architecture/components.md#base-components), and make sense add `StyledComponent` type to Props